superset/viz.py

```diff
@@ -1933,6 +1933,7 @@ class DeckScatterViz(BaseDeckGLViz):
     viz_type = 'deck_scatter'
     verbose_name = _('Deck.gl - Scatter plot')
     spatial_control_keys = ['spatial']
+    is_timeseries = True
```

Comment:
I was expecting to see something that turns `is_timeseries` on or off in the `query_obj` based on whether a `Time Grain` is selected. Eventually we'll hide the `Time Grain` control where `is_timeseries==False`

Reply:
Will do.